README.md

```diff
@@ -28,6 +28,26 @@ remotes::install_github("NOAA-FIMS/FIMS")
 library(FIMS)
 ```
 
+#### Fixing Fatal Error
```

Comment:
@Andrea-Havron-NOAA do you think we should move this to the "Windows" section below line 22 given it is a windows-specific problem?

Reply:
Agreed, it fits better under the Windows section